### PR TITLE
Step 8 & 9

### PR DESCRIPTION
**Pull request by taheryasir**
Step 8: Write a test (system spec)
- Get ready to write a spec
- Let's get prepared spec/spec_helper.rb, spec/rails_helper.rb
- Let's write a system spec for the task function

Step 9: Make various settings for the app
- Set the time zone
- Fix your time zone to Tokyo/Japan on Rails
- Configure error page
- Replace your customized error page from default
- Set the error page appropriately according to the situation
- Two types of status code settings, page 404 and page 500, are required at least

### CODE REVIEW

**Review by tana2034 — COMMENTED** [review]
I left a few comments 👍 please check🙂

File: myapp/spec/requests/tasks_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+require 'pp'
+
+RSpec.describe TasksController, type: :request do
+  before do
+    create(:task)
+    create(:task, name: 'task2', desc: 'just a sample description')
+  end
```
Comment (tana2034) [inline myapp/spec/requests/tasks_spec.rb:8]: These lines don't seem to be necessary.

**Review by tana2034 — COMMENTED** [review]
I checked request spec and looks fine!
and I left 1 comment, please check 🙇

File: myapp/spec/requests/tasks_spec.rb (hunk 1 above)
Comment (taheryasir) [reply myapp/spec/requests/tasks_spec.rb]: removed

**Review by tana2034 — APPROVED** [review]
LGTM!
LGTM

**Pull request merged**